doc/api/buffer.markdown

```diff
@@ -392,6 +392,19 @@ byte from the original Buffer.
     // abc
     // !bc
 
+
+### buf.indexOf(value[, byteOffset])
+
+* `value` String, Buffer or Number
+* `byteOffset` Number, Optional, Default: 0
+* Return: Number
+
+Operates similar to
+[Array#indexOf()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/indexOf).
```

Comment:
Yes there is an issue with the styling. If written as
```
[`Array#indexOf()`](...)
```
then there is no visual queue that the text is a link. So I intentionally didn't write it that way.